src/components/organisms/Form.tsx

```diff
@@ -1,14 +1,27 @@
-import React from "react";
+import React, { useState } from "react";
 import { styled } from "@stitches/react";
-import { indigo, mauve, blackA } from "@radix-ui/colors";
+import { indigo, mauve, blackA, tomato } from "@radix-ui/colors";
 import * as TabsPrimitive from "@radix-ui/react-tabs";
 import * as CheckboxPrimitive from "@radix-ui/react-checkbox";
 import { CheckIcon } from "@radix-ui/react-icons";
+import { useForm, SubmitHandler } from "react-hook-form";
+import Login from "@/components/molecules/Login";
 
 type Props = {
   setLoggedin: React.Dispatch<React.SetStateAction<boolean>>;
 };
 
+interface SignupFormInput {
+  firstname: string;
+  lastname: string;
+  email: string;
+  phone: string;
+  password: string;
+  serialnumber: string;
+  zipcode: string;
+  c1: boolean;
+}
```

Comment:
What is `c1`?

Reply:
Also don't include this file because this file is implementing sign up validation for #30.
Remove this file from this branch, make a new branch for #30 and move this file to a new branch.

Reply:
I can't remove this file because it includes login file, but I reverted the file before implementing validation.